Qast/Managers/FirebaseManager.swift

```diff
@@ -1,43 +1,50 @@
-import Foundation
-import Firebase
+//
+//  FirebaseManager.swift
+//  Qast
+//
+//  Created by Austin Welch on 7/5/19.
+//  Copyright © 2019 Qast. All rights reserved.
+//
+
 import FirebaseFirestore
 import CoreLocation.CLLocation
 
-let DB_BASE = Firestore.firestore()
-
-typealias SoundZoneCompletionBlock = ([SoundZone], Error?) -> Void
+struct FirebaseManager: SoundZoneAPI {
```

Comment:
So is the plan here to have a single FirebaseManager that will implement each repository protocol?
like:

`FirebaseManager: SoundZoneAPI, UserAPI, VenueAPI `

Reply:
We can make multiple managers if we'd like or, do something like you have there

Reply:
How about we just start with this monolithic FirebaseManager and have protocol extensions for each domain object API. So yeah this is fine